presto-parquet/src/main/java/com/facebook/presto/parquet/dictionary/DictionaryReader.java

```diff
@@ -36,11 +34,9 @@ public DictionaryReader(Dictionary dictionary)
     }
 
     @Override
-    public void initFromPage(int valueCount, byte[] page, int offset)
+    public void initFromPage(int valueCount, ByteBufferInputStream in)
```

Comment:
- `in` -> `inputStream` (we don't abbreviate names -- looks like this is old code, but please change it)
- add `requireNonNull(inputStream, "inputStream is null")`

Reply:
Done